src/less-style/question-props.less

```diff
@@ -157,3 +157,52 @@
   padding: 0 2em;
   margin-bottom: 2em;
 }
+
+.fake-input {
```

Comment:
Originally wanted this to inherit from styles in HQ. however that proved to be a little more involved than I thought. Would require editing the bootstrap files as we're on less 1.3 (was going to use: http://lesscss.org/features/#extend-feature) and compilation breaks when upgraded to 1.4